### PR TITLE
Add prow logs analysis promotion tweet

### PR DESCRIPTION
**Pull request by Shreyanand**
Add a tweet for promoting prow logs EDA notebook.

### CODE REVIEW

File: tweets/2021-07/prow-logs.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Interested in analyzing prow logs data available on GCS/origin-ci-test? Check out https://www.youtube.com/watch?v=JjFWFaMfUJA to learn how to fetch the logs and perform preliminary analysis.
```
Comment (oindrillac) [inline tweets/2021-07/prow-logs.tweet:1]:
```suggestion
Are you a Kubernetes or OpenShift developer interested in analyzing Prow logs? Check out https://www.youtube.com/watch?v=JjFWFaMfUJA to learn how to fetch Prow logs stored on GCS and perform preliminary analysis.
```
@Shreyanand How about?

**Review by oindrillac — APPROVED** [review]
/lgtm

**Review by aakankshaduggal — APPROVED** [review]
/lgtm

**Comment by Shreyanand** [comment]
I just squashed the commits :rofl:

**Pull request merged**